lib/SILOptimizer/Mandatory/DefiniteInitialization.cpp

```diff
@@ -1070,6 +1077,284 @@ void LifetimeChecker::injectActorHops() {
     injectExecutorHopAfter(point);
 }
 
+void LifetimeChecker::injectTypeWrapperStorageInitalization() {
```

Comment:
probably a refactoring TODO for the future is to move this function and the actor hop injection code out of this file and into some sort of `DICodeSynthesis.cpp` file.

Reply:
Will add!

Reply:
Added!